.github/ISSUE_TEMPLATE/other-issue.md

```diff
@@ -0,0 +1,6 @@
+---
+name: How-to and other issues
+about: Please, post other questions on StackOverflow with `gpflow` tag.
```

Comment:
"please, ..." with a comma means something slightly different to "please ..." without a comma. The former carries more of a weight on please, and suggests displeasure or annoyance.

more generally, I avoid the word. Either I reword the sentence without "please", or I leave out the sentence

@awav fyi since you added it above

Reply:
```suggestion
about: Please post "how do I do ..." questions on StackOverflow with `gpflow` tag.
```